src/btouch.cs

```diff
@@ -116,6 +116,8 @@ string GetAttributeLibraryPath ()
 				return Path.Combine (GetSDKRoot (), "lib", "bgen", "Xamarin.Mac-full.BindingAttributes.dll");
 			} else if (target_framework == TargetFramework.Xamarin_Mac_2_0_Mobile) {
 				return Path.Combine (GetSDKRoot (), "lib", "bgen", "Xamarin.Mac-mobile.BindingAttributes.dll");
+			} else if (target_framework == TargetFramework.DotNet_5_0_macOS) {
+				return Path.Combine (GetSDKRoot (), "lib", "bgen", "Xamarin.Mac-full.BindingAttributes.dll");
```

Comment:
That's the wrong path for .NET, this is the correct one:

```suggestion
				return Path.Combine (GetSDKRoot (), "lib", "Xamarin.Apple.BindingAttributes.dll");
```

and that's the right value for all platforms, so this can go at the top of the method, something like this:

```csharp
if (IsDotNet)
    return Path.Combine (GetSDKRoot (), "lib", "Xamarin.Apple.BindingAttributes.dll");
```